src/core/utils/callbacks.cpp

```diff
@@ -71,7 +72,8 @@ void CallbackVerbose::operator()(SolverAbstract& solver) {
   std::cout << solver.get_stop() << "  " << -solver.get_d()[1] << "  ";
   std::cout << solver.get_xreg() << "  " << solver.get_ureg() << "  ";
   std::cout << std::fixed << std::setprecision(4) << solver.get_steplength() << "  ";
-  std::cout << std::scientific << std::setprecision(precision_) << solver.get_ffeas();
+  std::cout << std::scientific << std::setprecision(precision_) << solver.get_ffeas() << "  ";
+  std::cout << std::scientific << std::setprecision(precision_) << solver.get_hfeas();
```

Comment:
We may want to also output the inequality infeasibility

Reply:
Now, we are printing the feasibility of the inequality constraints. I have to modify the log files as well.

Note that I created a function for computing the inequality feasibility.